src/CSET/_workflow_utils/parbake_recipe.py

```diff
@@ -31,35 +31,31 @@ def data_directories() -> list[str]:
 
 def run_recipe_steps():
     """Process data and produce output plots."""
+    print("Retrieving recipe from cookbook.")
     recipe = recipe_file()
-    data_dirs = data_directories()
-    # Construct the plot output directory for the recipe.
-    output_dir = f"{os.environ['CYLC_WORKFLOW_SHARE_DIR']}/web/plots/{os.environ['CYLC_TASK_ID']}"
 
-    command = (
-        ["cset", "bake", "--recipe", recipe, "--input-dir"]
-        + data_dirs
-        + ["--output-dir", output_dir]
-    )
-
-    colorbar_file = os.getenv("COLORBAR_FILE")
-    if colorbar_file:
-        command.append(f"--style-file={colorbar_file}")
-
-    plot_resolution = os.getenv("PLOT_RESOLUTION")
-    if plot_resolution:
-        command.append(f"--plot-resolution={plot_resolution}")
+    case_aggregation = bool(os.getenv("DO_CASE_AGGREGATION"))
+    data_dirs = data_directories(case_aggregation)
+    # Construct the location for the recipe.
+    if case_aggregation:
+        recipe_dir = f"{os.environ['ROSE_DATAC']}/aggregation_recipes/"
+    else:
+        recipe_dir = f"{os.environ['ROSE_DATAC']}/recipes/"
+    os.makedirs(recipe_dir, exist_ok=True)
+    output = f"{recipe_dir}{os.environ['CYLC_TASK_NAME']}.yaml"
 
-    skip_write = os.getenv("SKIP_WRITE")
-    if skip_write:
-        if skip_write == "True":
-            command.append("--skip-write")
+    # Construct command to run.
+    command = ["cset", "parbake", "--recipe", recipe, "--output", output]
+    # Append input data paths.
+    command += ["--INPUT_PATHS", str(data_dirs)]
+    # Append additional command line arguments.
+    command += shlex.split(os.getenv("CSET_ADDOPTS", ""))
 
     print("Running", shlex.join(command))
     try:
         subprocess.run(command, check=True)
     except subprocess.CalledProcessError as err:
-        print(f"cset bake exited with non-zero code {err.returncode}.", sys.stderr)
+        print(f"cset parbake exited with non-zero code {err.returncode}.", sys.stderr)
```

Comment:
```suggestion
        print(f"cset parbake exited with non-zero code {err.returncode}.")
```
This should have been `file=sys.stderr`, but I think it is more useful going to stdout anyway.